vault_contract/README.md

```diff
@@ -6,15 +6,13 @@ The Vault smart contract is a secure repository for safeguarding verifiable cred
 ## Features
 With this smart contract, you will be able to:
 
+- Initialize a vault by deploying a DID and storing the corresponding DID URI.
 - Authorize a list of issuers to store verifiable credentials in a vault.
 - Authorize an issuer to store verifiable credentials in a vault.
 - Revoke an issuer for a specific vault.
 - Store a verifiable credential in the recipient's vault.
-- Deploys a DID and registers a vault using the deployed DID.
-- Register a vault given its DID.
-- Revoke a vault given its DID.
-- Retrieve a vault given its DID.
-- List all vaults.
+- Revoke a vault.
+- Retrieve the list of stored vcs in the vault.
```

Comment:
In general, try to remove `in a vault` since now we will have just one Vault which is the contract itself. IMO we could remove this part of the sentence or specify `in the vault`.